app/views/catalog/_save_search.html.erb

```diff
@@ -8,7 +8,8 @@
         <h4 class="modal-title" id="save-modal-label"><%= t(:'spotlight.saved_search.label') %></h4>
       </div>
       <div class="modal-body">
-        <%=f.text_field :title, label: t(:'spotlight.saved_search.title'), required: true %>
+        <%= f.text_field :title, label: t(:'spotlight.saved_search.title') %>
+        <%= select_tag :id, options_for_select(current_exhibit.searches.map { |s| [s.title, s.id] }), include_blank: true, label: t(:'spotlight.saved_search.id') %>
```

Comment:
I'm getting a bit of odd behavior with these field locally.

<img width="616" alt="modal" src="https://cloud.githubusercontent.com/assets/96776/22713090/e8b14352-ed3b-11e6-9b2a-f9a63701b060.png">

<img width="677" alt="markup" src="https://cloud.githubusercontent.com/assets/96776/22713091/e8b1f568-ed3b-11e6-81d5-6b9a45fc50ab.png">

Perhaps `f.select` instead of `select_tag` could clear that up?